weeklyChallenges01/ChallengesWithTest01.cs

```diff
@@ -8,37 +8,64 @@ public class ChallengesSet01
     {
         public bool AreTwoNumbersTheSame(int num1, int num2)
         {
-            throw new NotImplementedException();
+            if ( num1 == num2)
+            {
+                return true;
+            }
+            else
+            {
+                return false;
+            }
         }
 
         public double Subtract(double minuend, double subtrahend)
         {
-            throw new NotImplementedException();
+            double difference = minuend - subtrahend;
```

Comment:
For optimization, maybe skip creating a variable to house the function and simply `return minuend - subtrahend;`